siac/cmds.go

```diff
@@ -47,20 +48,64 @@ func updatecmd(cmd *cobra.Command, args []string) {
 		cmd.Usage()
 		return
 	}
-	fmt.Println(getResponse("/update", &url.Values{
-		"action": {args[0]},
-	}))
+	switch args[0] {
+	case "check":
+		if m, err := getResponse("/update/check", nil); err == nil {
+			if m["Available"].(bool) {
+				fmt.Printf("Update %s is available!\nRun 'siac update %s' to install it.\n", m["Version"], m["Version"])
+			} else {
+				fmt.Println("Up to date!")
+			}
+		} else {
+			fmt.Println("Update check failed:", err)
+		}
```

Comment:
We haven't standardized on this, but I prefer the form:
```go
m, err = stuff
if err != nil {
    // control statement
}
// more stuff
```

The benefit being consistency. I like having `err != nil` be what always happens because then you get used to looking for it and looking at it, and because by doing `if err != nil { control }` you get to skip out on a bunch of chained `if`s for when there's lots of error checking.

Forest would definitely disagree here, he likes his flexibility, but I feel like being strict and consistent is important for having highly scalable and highly maintainable codebases.